services/web/server/src/simcore_service_webserver/resource_manager/garbage_collector.py

```diff
@@ -369,71 +430,37 @@ async def remove_orphaned_services(
         node_ids = await get_workbench_node_ids_from_project_uuid(app, project_uuid)
         currently_opened_projects_node_ids.update(node_ids)
 
-    running_interactive_services: List[
-        Dict[str, Any]
-    ] = await get_running_interactive_services(app)
-    logger.debug(
-        "Will collect the following: %s",
-        [x["service_host"] for x in running_interactive_services],
-    )
-    for interactive_service in running_interactive_services:
-        # if not present in DB or not part of currently opened projects, can be removed
-        node_id = interactive_service["service_uuid"]
-        service_host = interactive_service["service_host"]
-        # if the node does not exist in any project in the db, we can safely remove it without saving any state
-        if not await is_node_id_present_in_any_project_workbench(app, node_id):
-            logger.info(
-                "Will remove orphaned service without saving state since this service should is not part of any project %s",
-                service_host,
+    running_interactive_services: List[Dict[str, Any]] = []
+    try:
+        running_interactive_services = await get_services(app)
+    except Exception:  # pylint: disable=broad-except
+        logger.exception(
+            (
+                "Normally this is not an issue. GC could not recover services "
+                "from director-v2, see error for details."
```

Comment:
is this not already catched at a higher level? yes. see line 107. again I'm not sure the message brings a lot of information for debugging.

Reply:
You are right, the exception was too generic. Changed it to the correct type.

The message would be shown when director-v2 is failing. In that case I will not print anything. Will demote to debug and change message.